pkg/swarm/swarm.go

```diff
@@ -185,3 +185,24 @@ func (c *chunk) WithType(t Type) Chunk {
 type ChunkValidator interface {
 	Validate(ch Chunk) (valid bool)
 }
+
+type chunkValidators struct {
+	set []ChunkValidator
+	ChunkValidator
+}
+
+func NewChunkValidators(v ...ChunkValidator) ChunkValidator {
```

Comment:
this returns a `ChunkValidator` but the function is called `NewChunkValidators` (plural).

we should be consistent here, take a look at the calls:
```
validator := swarm.NewChunkValidators(validatormock.NewValidator(true))
chunkvalidators := swarm.NewChunkValidators(soc.NewValidator(), content.NewValidator())
mockValidator := swarm.NewChunkValidators(mock.NewValidator(true))
```

i think it's OK to have an entity called _Validator_ even if it uses multiple private validators internally, but the vars which take the result of this func (plus the func name) should have consistent naming in this manner.